New article "Detect languages with the Language Detector API" (#3736)
MicrosoftDocs/edge-developer: microsoft-edge/web-platform/translator-api.md

<!-- ====================================================================== -->
## Model availability

> An initial download of the model will be required the first time a website calls a built-in AI API.  You can monitor the model download by using the monitor option when creating a new Translator API session; see [Monitor the progress of the model download (monitor)](#monitor-the-progress-of-the-model-download-monitor), below.

Reviewer: ```suggestion
An initial download of the model is required the first time a website calls a built-in AI API.  You can monitor the model download by using the monitor option when creating a new Translator API session; see [Monitor the progress of the model download (monitor)](#monitor-the-progress-of-the-model-download-monitor), below.
```
per corp style, favor present tense
simple wording is sufficient
